src/App.tsx

```diff
@@ -1,18 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import './App.scss';
+import { Tabs } from './Tabs';
 
 export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const handleTabselected = (tab: Tab) => {
+    if (tab.id !== selectedTab.id) {
+      setSelectedTab(tab);
+    }
+  };
+
+  return (
+    <div className="App m-3">
+      <h1>
+        Selected tab is&nbsp;
+        {selectedTab.title}
+      </h1>
+
+      <Tabs
+        tabs={tabs}
+        selectedTab={selectedTab}
+        handleTabselected={handleTabselected}
+      />
+      <div data-cy="content m-5">{selectedTab.content}</div>
```

Comment:
Implement the Tabs component accepting tabs as a prop and displaying a button per each tab and **the content of the selected tab**
so move this to your `Tabs` component